packages/plugins/plugin-table/src/components/Table/ColumnActionsMenu.tsx

```diff
@@ -2,50 +2,37 @@
 // Copyright 2024 DXOS.org
 //
 
-import React, { type RefObject } from 'react';
+import React from 'react';
 
-import { DropdownMenu, type DropdownMenuRootProps, useTranslation } from '@dxos/react-ui';
+import { DropdownMenu, useTranslation } from '@dxos/react-ui';
 
 import { TABLE_PLUGIN } from '../../meta';
 import { type TableModel } from '../../model';
 
-export type ColumnActionsMenuProps = {
-  triggerRef: RefObject<HTMLButtonElement>;
-  model?: TableModel;
-  fieldId?: string;
-  onShowColumnSettings: () => void;
-} & Pick<DropdownMenuRootProps, 'open' | 'onOpenChange'>;
-
-export const ColumnActionsMenu = ({
-  triggerRef,
-  model,
-  fieldId,
-  open,
-  onOpenChange,
-  onShowColumnSettings,
-}: ColumnActionsMenuProps) => {
+export const ColumnActionsMenu = ({ model }: { model?: TableModel }) => {
```

Comment:
always prefer an exported type definition as a standard.
model prob shouldn't be optional; i don't think we're consistent about this, but i'd try to make things non-optional if the component wouldn't function without it (different for hooks, since we might not have one available and you can't have optional hooks). callbacks should all be optional.